.github/workflows/python.yml

```
@@ -1,13 +1,15 @@
 name: Python package
 on:
   push:
+    branches: [ main ]
```

Good catch! Let's run on tags too:
```suggestion
    branches:
      - main
    tags:
      - v*.*.*
```